Apicurio-Test-Plan.jmx

```diff
@@ -58,7 +58,7 @@
         </com.sngular.kloadgen.config.schemaregistry.SchemaRegistryConfigElement>
         <hashTree/>
         <com.sngular.kloadgen.config.valueserialized.ValueSerializedConfigElement guiclass="TestBeanGUI" testclass="com.sngular.kloadgen.config.valueserialized.ValueSerializedConfigElement" testname="KLG - Value Schema Serializer Config" enabled="true">
-          <stringProp name="valueNameStrategy">io.apicurio.registry.serde.avro.strategy.RecordIdStrategy</stringProp>
+          <stringProp name="valueNameStrategy">com.sngular.kloadgen.strategy.RecordSubjectNameStrategy</stringProp>
```

Comment:
Apicurio-Test-Plan uses the kloadgen strategy and Example-Test-Plan-2 uses the Apicurio strategy, is this correct?
It seems as if Apicurio-test-plan should use Apicurio's strategy instead.

Reply:
Functionality test in Apicurio-Test-Plan uses com.sngular.kloadgen.strategy.RecordSubjectNameStrategy as serializer's strategy